student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Take a look at the image below. When I try to instantiate a Student object I get the error as shown in the image. Kindly fix this. (Hint: `classroom` can be the only `initialize` parameter; `age` can be the only parameter for `super`)

  ![mmesay-stud](https://user-images.githubusercontent.com/36542986/145904993-b925f161-236d-4131-a2a9-7e47fb5695c1.jpg)